.gitignore

```diff
@@ -2,6 +2,10 @@
 *.swp
 .settings
 
+src/main/res/values/defaults.xml
+src/main/java/eu/siacs/conversations/Config.java
```

Comment:
```
src/main/res/values/defaults.xml
src/main/java/eu/siacs/conversations/Config.java
```

Adding these 2 lines are wrong. Because application need these files to run.

Reply:
removed

Reply:
Am I missing something? I see other lines are deleted from gitignore